Git lab add cancel pipeline command (#29772)
demisto/content: Packs/GitLab/ReleaseNotes/2_2_18.md

#### Integrations
##### GitLab v2
- Updated the Docker image to: *demisto/python3:3.10.13.74666*.

> - Added a new command ***gitlab-cancel-pipeline*** which allows to cancel a GitLab pipeline.

Reviewer: ```suggestion
- Added a new command ***gitlab-cancel-pipeline*** that enables canceling a GitLab pipeline.
```
